src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -36,7 +36,10 @@ namespace AppInstaller::CLI
             Execution::TableOutput<3> sourcesTable{ context.Reporter, { header, Resource::String::SourceListType, Resource::String::SourceListArg } };
             for (const auto& source : sources)
             {
-                sourcesTable.OutputLine({ source.Name, source.Type, source.Arg });
+                if (!source.IsHidden)
+                {
+                    sourcesTable.OutputLine({ source.Name, source.Type, source.Arg });
+                }

```

Comment:
Why do we want to hide the sources from the list of Group Policy sources? Is there a way to show them?
What about the list in `source list`?